src/components/Toast/README.md

```diff
@@ -91,23 +99,33 @@ Messages should be:
 - Product updated
 - Collection added
 - Customer updated
-- No internet connection
+- Internet disconnected
+- Connection timed out
 
 #### Don’t
 
+- No internet connection
+- Can’t charge negative tax rates
+- Your online store has a maximum of 20 themes. Delete unused themes to add more.
 - Your product has been successfully updated
 - We were unable to save the customer
 - Your Order was Archived Today
 - Discount: Saved successfully
 
 <!-- end -->
 
-### Action
+### Toast with action
+
+Only include an action in toast if the same action is available elsewhere on the page. For example:
+
+- If merchants need to reload a section, offer the call to action [Reload] in the toast. If they miss the toast message, they can also refresh the entire page.
+- If merchants delete an image, offer the option to [Undo] the deletion. If they miss it in the toast message, they can still retrieve it from somewhere else.
 
 Action should:
 
-- Keep the action label short. Preferably 1 verb
-- Not have actions for dismissing toast
+- Keep the action label short, preferably 1 verb.
+- Not have actions, like [Cancel], for dismissing toast. The [X] to dismiss is already included in the component.
+- Be used with a duration of at least 10,000 milliseconds for accessibility.
```

Comment:
This seems reasonable to enforce in code that toast with actions have a default of 10000ms. If an action is present and the duration is less than 10000ms we should at least log a warning and force the duration to 10000.

Reply:
Thank you! I had suggested this, but it seemed we shouldn't have one prop have a side effects on another prop.

Reply:
If no duration is provided, but an action is, then the default could be 10000ms instead of 5000ms so we don't override. Then only log a warning if the provided duration is less than 10k when an action is present. Not terribly opinionated either way, but I don't really see any downside to overriding a provided duration prop other than the principle that it is gross.

Reply:
Overriding a provided duration feels a bit heavy-handed, but a log message sounds like a great idea to me. Providing a default of 10,000ms if no duration is provided makes a lot of sense to me.